Sources/PitchSpeller/Data Structures/UndirectedEdge.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  UndirectedEdge.swift
+//  PitchSpeller
+//
+//  Created by Benjamin Wetherfield on 7/1/18.
+//
+
+public struct UndirectedEdge<Node, Num>: EdgeLike {
+    
+    typealias Weight = Num
+    typealias Pair = UnorderedPair<Node>
+    
+    var weight: Weight
+    let nodes: Pair
+    
+    init (_ source: Node, _ destination: Node, _ weight: Weight) {
+        self.nodes = Pair(source, destination)
+        self.weight = weight
+    }
+    
+    init (_ nodepair: Pair, _ weight: Weight) {
```

Comment:
I could imagine this being named `nodes` or `nodePair`.

Reply:
Yeah, I like `nodes`